### ISSUE TITLE
Track Changes: XML broken: The end tag w:ins does not match with the start tag w:rPr

### ISSUE THREAD

**Issue by Commifreak** [CollaboraOnline/online#5594]
**Describe the bug**
Sometimes, if track changes is enabled, collabora breaks the xml block for a change, resulting in xml error and broken document.
One can fix it to remove the not starting `w:ins` tag and repait the document but it happens again after a few edits

**To Reproduce**
No valid steps known :(

**Expected behavior**
No XML corruption

**Actual behavior**
This is one `w:pRr` block:

```xml
			<w:pPr>
				<w:pStyle w:val="Normal"/>
				<w:bidi w:val="0"/>
				<w:spacing w:lineRule="auto" w:line="252"/>
				<w:ind w:left="0" w:right="0" w:hanging="0"/>
				<w:jc w:val="both"/>
				<w:rPr>
					<w:del w:id="392" w:author="Username" w:date="2022-12-14T09:27:26Z"/>
				</w:ins>
			</w:rPr>
		</w:pPr>
```

You see, the `</w:ins>` has no starting tag which breaks the document. The document is then not editable anymore.

**Screenshots**
none

**Desktop (please complete the following information)**
 - OS: Ubuntu 20.04
 - Browser: Firefox
 - Version: 107

**Smartphone (please complete the following information)**
none

**Additional context**
CODE: 22.05.8.4.1 via docker

I dont know if there is another issue already - did not found anything.

**Comment by thebearon** [comment]
Please try to find an exact reproducer, we can't do anything about the problem until there is one.

**Comment by thebearon** [comment]
Probably the same issue as this: [tdf#147892](https://bugs.documentfoundation.org/show_bug.cgi?id=147892)

**Comment by Commifreak** [comment]
Indeed. Seem to be it

**Comment by thebearon** [comment]
Let's assume it is. The fix is now ported to co-22.05 as the following commit, and will be part of the next CODE release.
https://github.com/LibreOffice/core/commit/f91e083e2565f07a6ebd2f2c0aa69bbacb4cfd19
